### PR TITLE
Update apikey.md

### PR DESCRIPTION
**Pull request by heyitswei**
Updated docs for Create API Key

### CODE REVIEW

File: docs/apikey.md [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
-2. Choose Personal Settings
-3. Click the Create API Key button
-4. Set a description and choose an expiration date
-5. Save the API key securely in your computer. You are not going to retrieve the plain text key again in the console.
\ No newline at end of file
+You need to create an API key to access the Timeplus REST API. Here’s how to create one: 
+
+1. Click on your **avatar** in the top right corner. In the dropdown, click **Personal Settings**.
+
+<img width="1361" alt="api-key-avatar-1" src="https://github.com/timeplus-io/docs/assets/107260595/4aa9e9af-01e3-46e9-8724-aa10fde0da3e">
```
Comment (jovezhong) [inline docs/apikey.md:7]: should not use assets in this way. Either need to checkin the image or use a public available image.

File: docs/apikey.md [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
+
+2. In the API Keys section, click the **Create API Key** button.
+
+<img width="1362" alt="api-key-settings-2" src="https://github.com/timeplus-io/docs/assets/107260595/efa73b94-f7ba-4b1f-b7b8-7a1f3c347662">
```
Comment (jovezhong) [inline docs/apikey.md:11]: wrong

File: docs/apikey.md [hunk 3]
```diff
@@ -1,7 +1,19 @@
...
+
+3. Enter an optional description for the key, choose an expiration date if needed, and click **Create**.
+
+<img width="1362" alt="api-key-dialog-3" src="https://github.com/timeplus-io/docs/assets/107260595/17c60d79-1bb3-4978-b707-c2955b5fce29">
```
Comment (jovezhong) [inline docs/apikey.md:15]: wrong

File: docs/apikey.md [hunk 4]
```diff
@@ -1,7 +1,19 @@
...
+
+4. The API key will only be shown to you once - make sure you securely save it right away. You won’t be able to retrieve the key again later in the console.
+
+<img width="1362" alt="api-key-copy-4" src="https://github.com/timeplus-io/docs/assets/107260595/509addcf-bb89-423b-b311-299c36e1e561">
```
Comment (jovezhong) [inline docs/apikey.md:19]: wrong

**Review by jovezhong — CHANGES_REQUESTED** [review]
use markdown syntax for inserting image and upload the images if it's part of the repo, or use external image link, such as from timeplus.com

**Comment by heyitswei** [comment]
Closing this PR as these changes are duplicate

**Pull request closed without merging**